src/app/listing-page/listing-page.component.html

```diff
@@ -6,19 +6,18 @@
     </div>
     <div class="listing-text">
       <div class="listing-name">
-        <a href="#">{{listing.name}}</a>
+        <a href="localhost:4200/listing?listing={{listing}}">{{listing.name}}</a>
```

Comment:
I also mislead you a bit in my architecture overview a couple of weeks ago when I said to use query params for the listing view page link. For this page it would actually be easier and better convention to use path params:

`<a href="/listing/{{listing._id}}">{{listing.name}}</a>`

Reply:
Then you can use the router in the same way that i did for the `/verify` path:

` {path: 'verify/:token', component: VerificationPageComponent},`

For the listing view page it would be something like:

` {path: 'listing/:id', component: ListingViewComponent},`

Reply:
Then you can use the `ActivatedRoute` class like I did in the `VerificationPageComponent` to get the listing's id out of the path params in the url, so that you use this ID for a call to the back end to get the listing. 

Doing it this way will allow listings to be bookmarked or linked to directly.